src/components/Tabs.js

```diff
@@ -205,4 +128,184 @@ class TabWithoutRadium extends React.Component {
 
 const Tab = Radium(TabWithoutRadium);
 
+class MenuTabs extends StandardTabs {
```

Comment:
i've had issues with inheritance in the past as well, but i agree that it's important not to get too dogmatic - if you think it's an excellent use case that's 🆒 . i'll take a look as well